tests/integration/models/test_download_reports_models.py

```diff
@@ -67,3 +73,52 @@ def test_create_download_report(create_reports_db, db_session):
     assert rp.project_id_report == report.project_id_report
     assert rp.date_created == report.date_created
     assert rp.last_updated == report.last_updated
+
+
+def test_download_report_to_json_contains_size_and_count():
+
+    report = DataDownloadReport()
+
+    report.add_size_access_type("open", 100.0)
+    report.add_size_experimental_strategy("WXS", 101.0)
+    report.add_size_project_id("TCGA-YYY", 330)
+    report.add_count_access_type("closed", 3)
+    report.add_count_access_location("San Francisco, CA, USA", 303)
+
+    assert report.to_json() == json.loads(
+        """
+        {
+            "access_type_report": {
+                "open": {
+                    "downloaded_size_gb": 100.0,
+                    "user_interest_files_count": 0
+                },
+                "closed": {
+                    "downloaded_size_gb": 0,
+                    "user_interest_files_count": 3
+                }
+            },
+            "experimental_strategy_report": {
+                "WXS": {
+                    "downloaded_size_gb": 101.0,
+                    "user_interest_files_count": 0
```

Comment:
Would there be any benefit for testing additional permutations of default vs added sizes / counts for different experimental strategies (for example, another strategy with a default size / added count and third with added size and added count? And for the other reports as well?

Reply:
> Would there be any benefit for testing additional permutations of default vs added sizes / counts for different experimental strategies (for example, another strategy with a default size / added count and third with added size and added count? And for the other reports as well?

Instead, I just added a new test that exercises each `add_size/count_xxx` method in isolation and verifies the the JSON output is correct.